pkg/operator/controller/controller.go

```diff
@@ -71,19 +69,15 @@ const (
 
 var (
 	log = logf.Log.WithName("cdi-operator")
-
-	readyGauge = prometheus.NewGauge(
-		prometheus.GaugeOpts{
-			Name: monitoring.MetricOptsList[monitoring.ReadyGauge].Name,
-			Help: monitoring.MetricOptsList[monitoring.ReadyGauge].Help,
-		})
 )
 
 func init() {
-	metrics.Registry = prometheus.NewRegistry()
```

Comment:
the prometheus.NewRegistry() part happen here now it's expected https://github.com/kubevirt/containerized-data-importer/pull/3009/files#diff-9247a710772a8c06bdff51039c0ee47fa91dbdfcb0b23a0f08e23e1106f77551R9.

it's uses "github.com/machadovilaca/operator-observability/pkg/operatormetrics"

Reply:
I see, but we don't have an extra call to NewRegistry() as we did before, right?

Reply:
we do call twice since there is 2 calls to different metrics (like before):
1. https://github.com/kubevirt/containerized-data-importer/pull/3009/files#diff-e492805ba442ae8a4a1872f4650b03454bad344f0aa1b220f9015d1253a5beb3R75
2. https://github.com/kubevirt/containerized-data-importer/pull/3009/files#diff-4b24605fd1d75314b8b1e5c6b0f877fc5cdad0addb5ba3ed168ef798f1aca790R132

Reply:
and the NewRegistry() happen in here: https://github.com/machadovilaca/operator-observability/blob/992afaa58b0a9768606b590d83e3b3e6b81c2e91/pkg/operatorrules/registry.go#L7

Reply:
Ah, we also don't use the ctrl runtime's metrics pkg anymore `sigs.k8s.io/controller-runtime/pkg/metrics`, so I guess my initial concern may also be invalid

Reply:
Better safe then sorry (: